compiler/passes/errorHandling.cpp

```diff
@@ -285,6 +285,9 @@ bool ErrorHandlingVisitor::enterCallExpr(CallExpr* node) {
 
       VarSymbol* errorVar    = NULL;
       BlockStmt* errorPolicy = new BlockStmt();
+      Expr*      insert      = node->getStmtExpr();
+      if (insert == NULL)
+        insert = node;
```

Comment:
Right but I'm talking about the case when node->parentExpr == NULL and so insert == node. Then `insert->insertBefore()` might not be reasonable.

I'm not so concerned about `node->insertAtTail()`.